### DISCUSSION TITLE
Syntax Highlighting

### DISCUSSION THREAD

**Discussion by Beormund** [berry-lang/berry#212] [Ideas]
I took the liberty of registering the berry.tmLanguage.json grammar with [shiki](https://github.com/shikijs/shiki) an excellent syntax highlighter than can generate HTML from berry code. I hope that is OK. 

If there is no objection I plan to do the same with Pygments (used on mkdocs website) and Linguist (used by github). It seems to me assigning Python to berry code blocks to get partial highlighting simply inflates python usage on Github and only partially highlights berry code.  What do you think?

<img width="840" alt="Screenshot 2021-12-15 at 22 20 25" src="https://user-images.githubusercontent.com/75735592/146274010-28b10f69-17a6-45e4-b9f3-dadfd313ef0d.png">

<img width="951" alt="Screenshot 2021-12-15 at 22 41 05" src="https://user-images.githubusercontent.com/75735592/146276145-5687f8c6-a57b-4f5c-92e1-b17e8f650183.png">

**Comment by skiars** [comment]
Great! Of course we want to make Berry highlight correctly as much as possible, but unfortunately we have to pretend to be another language 😔.

**Comment by Beormund** [comment]
Some syntax highlighters (e.g., Rouge used by GitHub Pages/Jekyll) can disambiguate the language by using a shebang. Have you thought about standardising on a shebang for berry?  Maybe:

#!berry-lang

Thoughts?
